layers/+tools/shell/README.org

```diff
@@ -30,14 +38,51 @@ This layer configures the various shells available in Emacs.
 To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =shell= to the existing =dotspacemacs-configuration-layers= list in this
 file.
+** Install vterm (EXPERIMENTAL)
+=vterm= is the latest addition to Emacs' set of terminal emulators and the only
+one to be implemented in C, leveraging =libvterm=. It is the only one in Emacs at
+the moment to be as fast as a standalone terminal and to support =ncurses=, =vim=,
+=top=, =htop= and the likes.
+*** Install libvterm
+**** MacOS
+#+BEGIN_SRC shell
+brew install libvterm
+#+END_SRC
+
+**** GNU/Linux
+In Ubtuntu:
+
+#+BEGIN_SRC shell
+sudo apt-get install libvterm
+#+END_SRC
+
+In other distros, find it and install it in your favorite package manager.
+
+**** Windows
+Not supported at the moment, but [[https://github.com/akermu/emacs-libvterm/issues/12][but possibly coming up]].
+
+*** Clone emacs-libvterm
+#+BEGIN_SRC shell
+  git clone https://github.com/akermu/emacs-libvterm.git ~/.emacs.d/private/local/emacs-libvterm
+#+END_SRC
+
+*** Add it to your additional packages
```

Comment:
This section seems to be outdated since it's on Melpa now. https://melpa.org/#/vterm